src/Formatter/Json.php

```diff
@@ -38,20 +38,20 @@ class Json extends General
      *
      * @param array $event containing event data.
      *
+     * @psalm-suppress ImplementedReturnTypeMismatch
      * @return string
      *
      * @throws Exception
      */
     public function format($event)
     {
         $data = $this->getEventData($event);
+        $encoded = json_encode($data);
```

Comment:
Does it work the same as `try catch` block?

Reply:
The try catch was complaining that there was nothing to catch, because `json_encode` returns false if it fails, rather than throwing an error. So this should replicate the intended behaviour of throwing an error on fail. The only bit that's missing is the stacktrace, which I wasn't sure if I could replicate, or if I even needed to because the error should generate it's own stacktrace, shouldn't it?

Reply:
In that case its fine imo. Thanks for clarifying